src/main/target/AIKONF4/target.h

```diff
@@ -0,0 +1,143 @@
+/*
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "AIK4"
+#define USBD_PRODUCT_STRING "AIKONF4"
+
+#ifdef OPBL
+#define USBD_SERIALNUMBER_STRING "0x8020000" // Remove this at the next major release (?)
```

Comment:
Are there plans to release an OPBL version for this? If not this is not needed.

Reply:
No plans, will remove it